### PR TITLE
Adds more food recipes

### PR DESCRIPTION
**Pull request by Alberyk**
Adds some food recipes, sprites from bay, vore and tg.
Also, allows you to pick up corgis, with held corgi sprites from baystation.

### CODE REVIEW

File: code/modules/food/recipes_microwave.dm [hunk 1]
```diff
@@ -122,21 +122,28 @@ I said no!
...
+
+
 /datum/recipe/waffles
 	reagents = list("sugar" = 10)
 	items = list(
 		/obj/item/weapon/reagent_containers/food/snacks/dough,
 		/obj/item/weapon/reagent_containers/food/snacks/dough
 	)
-	result = /obj/item/weapon/reagent_containers/food/snacks/waffles
+	result = /obj/item/weapon/reagent_containers/food/snacks/classichotdog
```
Comment (Arrow768) [inline code/modules/food/recipes_microwave.dm]: Was that change intentional ?

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -4538,3 +4537,99 @@
...
+	desc = "A traditional human dish, recreated using space carp."
+	icon_state = "sashimi"
+	filling_color = "#FFDEFE"
+	center_of_mass = list("x"=17, "y"=13)
+
+/obj/item/weapon/reagent_containers/food/snacks/sashimi/Initialize()
+	. = ..()
+	reagents.add_reagent("seafood", 3)
+	reagents.add_reagent("carpotoxin", 3)
+	src.bitesize = 4
```
Comment (Arrow768) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: dont think the src. is needed here

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 3]
```diff
@@ -4538,3 +4537,99 @@
...
+	name = "honey bun"
+	desc = "A sticky pastry bun glazed with honey."
+	icon_state = "honeybun"
+	nutriment_desc = list("pastry" = 1)
+	nutriment_amt = 3
+
+/obj/item/weapon/reagent_containers/food/snacks/honeybun/Initialize()
+	. = ..()
+	reagents.add_reagent("honey", 3)
+	src.bitesize = 3
```
Comment (Arrow768) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Same here. src.

File: code/modules/food/recipes_microwave.dm (hunk 1 above)
Comment (Alberyk) [reply code/modules/food/recipes_microwave.dm]: Not, that was an accident, I will fix.
